data/sql/updates/pending_db_world/rev_1645574957481593300.sql

```diff
@@ -0,0 +1,1705 @@
+UPDATE `quest_template_locale` SET `Details`="午夜的时候，我们听到了吵闹的声音。今天早上我才发现羊圈里少了两只羊，那头名叫瓦加什的野兽袭击了我们的牲畜。由于国王麦格尼的军队大都在遥远的战场上与联盟其他的种族并肩作战，所以没有人能对付得了瓦加什。$b$b也许你有勇气找到这只猛兽并杀了它。把它的一颗牙齿交给我作为证明，我会奖赏你的。瓦加什就躲在这里的农场附近，小心点，它非常凶残。" WHERE `ID`=314 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Title`="<UNUSED>" WHERE `ID`=316 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="我正在准备为攻城坦克旅执行一项任务。这次任务会持续很长时间，所以我得预先收集能够维持一个月的物资。贝尔丁正忙着修理我的坦克，你能帮我去弄一些东西来吗？$B$B我需要一些毛皮来做床垫，还要一些野猪肉作为食品储备。猪肉当然是从野猪身上割下来的，最好的毛皮是熊皮……在灰色洞穴南部的雪地上到处都是这两种动物的踪迹。" WHERE `ID`=317 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="这附近的海滩上的船只残骸大多是属于库尔提拉斯第三舰队的，在上一次大战中，那支舰队曾与红龙们在此激战，不幸全军覆没了。$B$B那支舰队里的船大多是洛丹伦的战舰，但有一艘船却不是，那是一艘名叫“飞翔鱼鹰”的精灵驱逐舰。它运着满满一船的光铸铁加入了第三舰队的船队中，本以为这样就足够安全了，结果却讽刺性地落得个被红龙击沉的厄运……$B$B故事就讲到这儿吧，飞翔鱼鹰的残骸就在米奈希尔港的南边。如果你能在那里找到光铸铁的话，就把它给我带回来吧。" WHERE `ID`=321 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="你好不容易把那块光铸铁锭从泡在水里的箱子中拿了出来，显然，是鱼人偷走了其余的铁锭。猎杀飞翔鱼鹰号残骸周围的鱼人，拿到它们身上带着的光铸铁锭。" WHERE `ID`=324 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="维沙克公爵最近正在贵族议会中兴风作浪，他似乎知道一些我们的事情，而且还游说其他议员们，要他们通过他提出的对这里进行一次全面调查的提案。我有一个计划，能让维沙克迷上些别的什么东西……但我需要你的帮助。$b$b在夜色镇郊区的斯塔文的花园里生长着一种名叫蒂罗亚之泪的白色小花，去给我弄一朵来。另外我还需要一些麝鼠根，那种东西只生长在湿地，就在巨石水坝下的瀑布底部。" WHERE `ID`=335 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="第一章实在是太精彩了，令人欲罢不能！奈辛瓦里设定了小说的背景，而且在小说里提到了他的同伴们，甚至包括我！$b$b不过，糟糕的是……有几页好象弄丢了。似乎第1页，第4页，第6页和第8页落在丛林中的什么地方了！" WHERE `ID`=339 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="我正要作一次关于“坚韧的灵魂”的演讲，我想引用一本关于金属学的著作《金属的延展性》里的一些段落。暴风城堡里的皇家图书馆里应该有这本书的副本。你能把它拿来给我吗？如果可以的话，圣光定会保佑你的！$B$B皇家图书馆很大，但首席管理员米尔顿·西弗对每一个书架上有些什么书都了如指掌。告诉他我需要什么，他就会把书找给你的。" WHERE `ID`=343 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="我也想把《金属的延展性》这本书尽快抄完，但是……$B$B你看看，我只用自己的墨水抄写书籍。这墨水是用我自己的配方制作的，它不容易化开，也不容易弄脏纸，唯一的缺点就是它干得太快了！我的最后一瓶墨水在我抄完之前就用光了，而且我也没法去做新的墨水！$B$B要想再配点墨水的话，我就得先弄到一些赤脊山的瑞斯班矿石，一直以来，这种矿石都是由湖畔镇的工头奥斯洛为我提供的。" WHERE `ID`=345 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="我病了…病得……非常厉害……$b$b我…染上了…荆棘谷热疫…致死的疾…疾病……$b$b……只…只有吃下…野人莫…莫…莫克…的心脏…才…才有救…那…是…一只残忍而…狡猾的…野兽。$b$b……巫医安巴格瓦巫师…是…唯一会…召唤…它…的人……$b$b找到…住在南边…海岬…洞穴…里的安巴格瓦……$b$b……快点…我就要死了…就要死了……" WHERE `ID`=348 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Objectives`="找到巫医安巴格瓦并要求他召唤出野人莫克，然后将野人莫克的心脏交给芬·菲兹拉克。" WHERE `ID`=348 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Title`="<UNUSED>" WHERE `ID`=352 AND `locale` = "zhCN";
+UPDATE `quest_template_locale` SET `Details`="Great Mammoda. Enormous, she was, with claws and teeth like swords. Now, I was caught off my guard, but I don't think being ready would have helped my cause much. One swipe and poof! Darkness. They say I'm lucky to walk now...$b$bWell, not much enjoyment to be had in hearing that old story now, is there? Anyways, $n, if you'd like to try yourself against her, the Lady of the Loch," WHERE `ID`=352 AND `locale` = "zhCN";
```

Comment:
This is still English

Reply:
The English descriptions are correct. They are unused quests where that is the actual retail translation because not used - see https://cn.classic.wowhead.com/quest=352/unused.